pkg/mimir/modules.go

```diff
@@ -1034,6 +1037,23 @@ func (t *Mimir) initUsageStats() (services.Service, error) {
 	return t.UsageStatsReporter, nil
 }
 
+func (t *Mimir) initContinuousTest() (services.Service, error) {
+	start := func(ctx context.Context) error {
```

Comment:
Doesn't seem necessary to create a `start` method here for the service. Should be fine to just execute this as regular code here and pass `nil` for the start method